contactaccessrights/js/src/access-rights/models/location.js

```diff
@@ -0,0 +1,14 @@
+define([
+  'access-rights/modules/models',
+  'access-rights/services/location',
+  'common/services/api',
+  'common/models/model'
+], function (models) {
+  'use strict';
+
+  models.factory('Location', ['Model', 'api.location', function (Model, api) {
+    return Model.extend({
+      getAll: api.query.bind(api)
```

Comment:
`api.location.query` gets the `OptionValues` of a given group, couldn't have you used [`OptionGroup.valuesOf`](https://github.com/civicrm/civihr/blob/develop/org.civicrm.reqangular/src/common/models/option-group.js)?
```js
getAll: function() {
  return OptionGroup.valuesOf('hrjc_location');
}
```
or it's not that useful in your case?
_(same goes for the `Region` model)_